### ISSUE TITLE
[Feature Request (long term)] Light up systems based on statistics (like the ingame map)

### ISSUE THREAD

**Issue by Ligraph** [exodus4d/pathfinder#225]
This would probably be hard to implement, but it would be amazing if I could make systems light up based on different statistics, like the ingame map.  For example, if I used "Gas sigs" it would light up systems with lots of gas sigs brightly, and ones with few less brightly.  

I would prefer this as a client-side thing so different people can view different things, but I have no clue if its viable.

**Comment by exodus4d** [comment]
Yeah, this is a good "nice to have" feature, nothing on top of my *TODO* list ;).
The implementation is not difficult.
